test/e2e/case1_framework_deployment_test.go

```diff
@@ -365,6 +361,52 @@ var _ = Describe("Test framework deployment", func() {
 				g.Expect(arg).ToNot(Equal("--disable-spec-sync=true"))
 			}
 		}, 60, 5).Should(Succeed())
+
+		By(logPrefix + "cleaning up")
+
+		Kubectl("annotate", "-n", cluster.clusterName, "-f", case1ManagedClusterAddOnCR, "policy-addon-pause=true")
+
+		// This is hacky but this sets the ManifestWork to orphan everything so that we can remove the
+		// policy.open-cluster-management.io/sync-policies-on-multicluster-hub annotation and not have it delete
+		// the cluster namespace. This will get reset to SelectivelyOrphan when the addon controller is reconciled
+		// below.
+		Eventually(func(g Gomega) {
+			mw := GetWithTimeout(clientDynamic, gvrManifestWork, case1MWName, cluster.clusterName, true, 15)
+
+			err := unstructured.SetNestedField(mw.Object, "Orphan", "spec", "deleteOption", "propagationPolicy")
+			g.Expect(err).ToNot(HaveOccurred())
+
+			_, err = cluster.clusterClient.Resource(gvrManifestWork).Namespace(cluster.clusterName).Update(
+				context.TODO(), mw, metav1.UpdateOptions{},
+			)
+			g.Expect(err).ToNot(HaveOccurred())
+		}, 30, 5).Should(Succeed())
+
+		Kubectl(
+			"annotate",
+			"ManagedCluster",
+			cluster.clusterName,
+			"policy.open-cluster-management.io/sync-policies-on-multicluster-hub-",
+		)
+
+		Kubectl("annotate", "-n", cluster.clusterName, "-f", case1ManagedClusterAddOnCR, "policy-addon-pause-")
```

Comment:
Annotations are usually key and value aren't they?

Reply:
This syntax removes the annotation.